src/mlpack/methods/preprocess/preprocess_imputer_main.cpp

```diff
@@ -32,7 +32,9 @@ PROGRAM_INFO("Impute Data", "This utility takes a dataset and converts user "
 PARAM_STRING_IN_REQ("input_file", "File containing data,", "i");
 PARAM_STRING_OUT("output_file", "File to save output", "o");
 PARAM_STRING_IN("missing_value", "User defined missing value", "m", "");
-PARAM_STRING_IN("strategy", "imputation strategy to be applied", "s", "");
+PARAM_STRING_IN("strategy", "imputation strategy to be applied. Strategies "
+    "should be one of 'custom', 'mean', 'median', and 'listwise_deletion'.", "s"
+    , "");
```

Comment:
If the `"s",` is too wide for the previous line, can you put it on the next line with the comma attached, instead of breaking the line on the comma?